docs/best-practices/error-handing.md

```diff
@@ -1,3 +1,22 @@
+# Table of Contents
```

Comment:
Can we rename this file to `error-handling.md`? Will need to update the link in the `docs/best-practices.md`